### PR TITLE
improve Android code integration documentation

### PR DESCRIPTION
**Pull request by slorber**

### CODE REVIEW

**Comment by slorber** [comment]
can be closed after merge: https://github.com/tinycreative/react-native-intercom/issues/111

**Comment by henrikra** [comment]
Great improvements 👍

**Comment by henrikra** [comment]
I would put xmlns:tools="http://schemas.android.com/tools" inside of example code of AndroidManifest so you won't miss it easily :)

File: README.md [hunk 1]
```diff
@@ -50,8 +50,31 @@ Initialize Intercom in your `AppDelegate.m`
...
-```
-Intercom.initialize(getApplicationContext(), "your api key", "your app id");
+Your Android Application should look like:
+
+```java
+// ...
+import com.robinpowered.react.Intercom.IntercomPackage;
+import io.intercom.android.sdk.Intercom;
+
+public class MainApplication extends MultiDexApplication {
```
Comment (henrikra) [inline README.md]: What is this `MultiDexApplication`?

Comment (slorber) [reply README.md]: Hmmm actually I don't know, it's what you get when you eject from Expo (I think multidex is used for large Android applications)

**Comment by slorber** [comment]
@henrikra thanks for the reveiw I'e just made some changes

File: README.md [hunk 2]
```diff
@@ -50,27 +50,68 @@ Initialize Intercom in your `AppDelegate.m`
...
 ```xml
-<service
-     android:name="com.robinpowered.react.Intercom.IntercomIntentService"
-     android:exported="false">
-     <intent-filter
-        android:priority="999">
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.myapp"
+	  ...
+          xmlns:tools="http://schemas.android.com/tools"
```
Comment (henrikra) [inline README.md]: I would add comment  like `<!-- Add this line -->`

Comment (slorber) [reply README.md]: it wouldn't be valid xml comment imo so I prefered to put "..."

You want something like that? as you can see it breaks syntax highlighting on github so I prefered ...

```xml
<manifest package="com.myapp"
          <!-- add this line -->
          xmlns:tools="http://schemas.android.com/tools"
    >

   <!-- comment -->
</manifest>
```

Comment (henrikra) [reply README.md]: What ever works the best :D Point is that there is comment pointing what you need to add

Comment (slorber) [reply README.md]: this could be fine like this:

image

**Comment by mnogueron** [comment]
Really useful PR, hope that it will be merge soon for others! :)

**Pull request merged**